src/components/BigButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+interface BigButtonProps {
+  color?: string;
+  onChange?: () => void;
```

Comment:
We don't need to add an `onChange` prop here, since the `BigButton` builds off of a regular HTML button (signified by `styled.button`), it already has `onChange` as a possible prop, so we don't need to (and can't, in styled components) handle that!

In the future, it's useful to keep in mind that props to a styled component is meant to be help decide how to style. In the case of `onChange`, it won't help us decide how to style this button (i.e. what color, what padding, etc etc)

Reply:
Agree! Also, since we're only passing in the one color prop, I think we can go ahead and get rid of this interface. Instead, your styled component would look something like:

```
const BigButton = styled.button<{ $color?: string }>`
```